exercises/01-react-stateful/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Wouldn't it be better to have a single `tsconfig.json` with using https://www.typescriptlang.org/tsconfig#extends?

Reply:
I've created the base configuration file, see 10af64cbb04907b180197b4698f6f3ad63306bdc